test/e2e-tests.sh

```diff
@@ -28,11 +28,98 @@ header "Setting up environment"
 install_pipeline_crd
 install_dashboard_backend
 
-failed=0
+# failed=0
```

Comment:
This is commented out but later on we use it (( failed )), so does this definitely work?